packages/react-components/src/components/styles/ReactionOverlay.style.ts

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(reaction) */
+import { keyframes, memoizeFunction } from '@fluentui/react';
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * Generate random float between two numbers, including min and max
+ * @private
+ */
+export function getRandomFloat(minValue: number, maxValue: number): number {
+  return minValue + Math.random() * (maxValue - minValue);
+}
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * Generate random int between two numbers, including min and max
+ * @private
+ */
+export function getRandomInt(minValue: number, maxValue: number): number {
+  return Math.floor(getRandomFloat(minValue, maxValue + 1));
```

Comment:
Doesn't `maxValue + 1` mean it could return a value larger than the max? (Would be extermely unlikely, getRandomFloat would have to return the max value, but if it did then this would return maxValue + 1)

Reply:
`export function getRandomFloat(minValue: number, maxValue: number): number {
  return minValue + Math.random() * (maxValue - minValue);
}`

Math.random() provides a value greater equal 0 but less than 1. 
So we are doing max + 1  and then floor to make sure it is within our intended range.